src/site/content/en/learn/pwa/windows/index.md

```diff
@@ -145,7 +145,12 @@ To reduce phishing within PWAs, some desktop browsers may have additional measur
 
 An experimental capability, known as *tabbed mode*​ will let your PWA have a tab-based design similar to a web browser. In this case, the user can have several tabs opened from the same PWA but all tied together in the same operating system window, as you can see in the following image:
```

Comment:
Might wanna update this, since it's no longer just an image:

```suggestion
An experimental capability, known as *tabbed mode*​ will let your PWA have a tab-based design similar to a web browser. In this case, the user can have several tabs opened from the same PWA but all tied together in the same operating system window, as you can see below:
```